karma.conf.js

```diff
@@ -26,8 +26,7 @@ module.exports = function(config) {
     browsers: ['ChromeHeadless'],
     files: ['dist/test/index.js'],
     frameworks: ['mocha'], // use the mocha test frameworkl
-    plugins: ['karma-chrome-launcher', 'karma-mocha', 'karma-sourcemap-loader'],
-    reporters: ['dots'], // report results in these formats.
+    reporters: ['dots', 'coverage'], // report results in these formats.
```

Comment:
note to reviewers — you don't need to use the `plugins` field in karma.conf because karma knows to look for anything that's `karma-*` in node_modules.